### PR TITLE
testing/supercronic: new aport

### PR DESCRIPTION
**Pull request by kmmndr**
https://github.com/aptible/supercronic
Cron for containers

### CODE REVIEW

File: testing/supercronic/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+pkgver="0.1.9"
+pkgrel=0
+pkgdesc="Cron for containers"
+url="https://github.com/aptible/supercronic"
+arch="all"
+license="MIT"
+depends=""
+makedepends="go bash dep python"
+install=""
+subpackages=""
```
Comment (Cogitri) [inline testing/supercronic/APKBUILD]: You can remove the empty variables

File: testing/supercronic/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+export GOPATH="$srcdir"
+
+prepare() {
+	mkdir -p "${builddir%/*}"
+	mv "$srcdir"/$pkgname-$pkgver "$builddir"
+	default_prepare
+}
+
+build() {
+	cd "$builddir"
```
Comment (Cogitri) [inline testing/supercronic/APKBUILD]: You don't need to do `cd $builddir` anymore, abuild does that for you.

File: testing/supercronic/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+pkgrel=0
+pkgdesc="Cron for containers"
+url="https://github.com/aptible/supercronic"
+arch="all"
+license="MIT"
+depends=""
+makedepends="go bash dep python"
+install=""
+subpackages=""
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/aptible/$pkgname/archive/v${pkgver}.tar.gz"
```
Comment (Cogitri) [inline testing/supercronic/APKBUILD]: 1) You don't have to use curly braces here
2) Please replace the $pkgname variable in the url with the actual name, that way it's easier to paste it into the browser :)

**Comment by kmmndr** [comment]
Thank you very much @Cogitri for your review :-)

**Comment by kmmndr** [comment]
I just realized that I would have done better by adding more commits and squashing them later, after review

**Comment by Cogitri** [comment]
Directly amending to your initial commit is fine too

**Pull request closed without merging**